scripts/perftests/PerformanceTestRunner.ps1

```diff
@@ -0,0 +1,78 @@
+Param(
+    [Parameter(Mandatory = $true)]
+    [string]$resultsDirectoryPath,
+    [string[]]$nugetClients,
+    [string]$testDirectoryPath,
+    [string]$logsDirectoryPath,
+    [switch]$SkipCleanup
+)
+
+    . "$PSScriptRoot\PerformanceTestUtilities.ps1"
+
+    if (![string]::IsNullOrEmpty($testDirectoryPath) -And $(GetAbsolutePath $resultsDirectoryPath).StartsWith($(GetAbsolutePath $testDirectoryPath))) 
+    {
+        Log "$resultsDirectoryPath cannot be a subdirectory of $testDirectoryPath" "red"
+        exit(1)
+    }
+    
+    if ([string]::IsNullOrEmpty($testDirectoryPath)) 
+    {
+        $testDirectoryPath = $([System.IO.Path]::Combine($env:TEMP, "np"))
```

Comment:
There are a lot of outer `$()` in this PR that appear to be superfluous.  This works just fine for me and is more readable:

    $testDirectoryPath = [System.IO.Path]::Combine($env:TEMP, "np")

Also, what does `np` stand for?  How about "nugetperf"?

Reply:
np stands for nugetperf. 
I wanted as short of a name as possible to minimize potential long path problems.